product_cost_security/tests/test_product_cost_security.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2023 Onestein (<https://www.onestein.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import Form, TransactionCase
+
+
+class TestProductCostSecurity(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
```

Comment:
You can create an environment with tracking disable.
cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))

Reply:
Made the change